tests/acceptance/pageObjects/FilesPageElement/publicLinksDialog.js

```diff
@@ -1,25 +1,244 @@
+const assert = require('assert')
+const util = require('util')
+
 module.exports = {
   commands: {
+    /**
+     * function clicks the edit button from public link dialog box
+     * @param linkName Name of the public link
+     * @returns {Promise<void>}
+     */
+    clickLinkEditBtn: async function (linkName) {
+      const linkRowEditButtonSelector = util.format(this.elements.publicLinkEditButton.selector, linkName)
+      const linkRowEditButton = {
+        locateStrategy: 'xpath',
+        selector: linkRowEditButtonSelector
+      }
+      await this
+        .initAjaxCounters()
+        .waitForElementVisible(linkRowEditButton)
+        .click(linkRowEditButton)
+        .waitForOutstandingAjaxCalls()
+    },
+    /**
+     * function sets selectors for provided year, month and day for the public link expiration date
+     * @param year
+     * @param month
+     * @param day
+     * @returns {[{locateStrategy: string, selector: *}, {locateStrategy: string, selector: *}, {locateStrategy: string, selector: *}]}
+     * returns array of three selectors
+     */
+    setExpiryDateSelectorsXpath: function (year, month, day) {
```

Comment:
Split this into smaller functions, each for `year`, `month` and `day`

Reply:
:ok: